### DISCUSSION TITLE
error when trying to get qasm

### DISCUSSION THREAD

**Discussion by embersax** [cda-tum/mqt-qecc#332] [Q&A]
Hi, I was having the following errors when I tried to convert bv_n14 from QASMbench(https://github.com/pnnl/QASMBench) with the following script: 
ecc_qiskit_wrapper -f /.../bv_n14.qasm -ecc Q9Surface -fq 50 -m D -e "./bv_n14_Q9Surface.qasm"
and got error 
ValueError: Gate x not supported to encode in error code Q9Surface!

but read in the document that x gate is supported with  Q9Surface, and I tried to replace x with hzh and that also gives the same error.

**Comment by burgholzer** [comment]
Hi, thanks for opening this discussion.
Seems like either a documentation mistake or some kind of bug to me.
@lucasberent @pehamTom could someone of you please take a look at this?

**Reply by embersax** [reply]
Hi， 
Thanks for replying, the documentation looks like this and this is why I think x gate will work with Q9Surface

Best [
![fig](https://github.com/user-attachments/assets/a149c9de-9829-4b26-9737-83ad4f52efa5)
](url)

**Reply by lucasberent** [reply]
Hi thanks for raising this issue X should be supported as you say so this looks like a bug. We will look into this (note that, however, this part of the package is somewhat deprecated so it might take some time sorry for that).
